### PR TITLE
Add web-scale fizzbuzz.

### PR DESCRIPTION
**Pull request by kunaldes**
Calls into my Rust fizzbuzz service to determine fizzbuzzness of the input number.

### CODE REVIEW

File: src/commands/fizzbuzz.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  static makeFizzBuzzCall(number) {
+    return fetch(`https://fozzbizz.herokuapp.com/fizzbuzz?num=${number}`)
+      .then(response => {
+        if (response.ok) {
+          return response.json();
+        }
+
+        throw response.statusText;
+      }).then(fizzBuzzJSON => {
+        if (fizzBuzzJSON.is_fizz && fizzBuzzJSON.is_buzz) {
```
Comment (IanMitchell) [inline src/commands/fizzbuzz.js]: If you wanna be super fun, could do something like:

```javascript
return `${fizzBuzzJSON.is_fizz && "Fizz"}${fizzBuzzJSON.is_buzz && "Buzz"}`;
```

**Comment by kunaldes** [comment]
It's ok, I hate me too.

File: src/commands/fizzbuzz.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        }
+        if (fizzBuzzJSON.is_buzz) {
+          return 'Buzz';
+        }
+        return number;
+      });
+  }
+
+  message(msg) {
+    const input = Aquarius.Triggers.messageTriggered(msg, new RegExp('^fizzbuzz ([\\d]+)$', 'i'));
```
Comment (IanMitchell) [inline src/commands/fizzbuzz.js]: If you want, could use a raw regex bit here instead of a RegExp object

File: src/commands/fizzbuzz.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    if (input) {
+      Aquarius.Loading.startLoading(msg.channel);
+
+      this.makeFizzBuzzCall(input[1])
+        .then(finalString => {
+          msg.channel.send(finalString);
+          Aquarius.Loading.stopLoading(msg.channel);
+        }).catch(errorString => {
+          msg.channel.send(`unable to fizzbuzz. Failed with ${errorString}. Ping @IanMitchel1 on twitter with your complaints`);
```
Comment (IanMitchell) [inline src/commands/fizzbuzz.js]: oh my GOD

File: src/commands/fizzbuzz.js (hunk 1 above)
Comment (kunaldes) [inline src/commands/fizzbuzz.js]: you fell into the obvious fizzbuzz trap lol.

You have to return the number if it's neither fizz nor buzz.

Comment (IanMitchell) [inline src/commands/fizzbuzz.js]: Meant like under this if - change it to an or, put the one line return

File: src/commands/fizzbuzz.js [hunk 4]
```diff
@@ -39,7 +39,7 @@ class FizzBuzz extends Aquarius.Command {
   }
 
   message(msg) {
-    const input = Aquarius.Triggers.messageTriggered(msg, new RegExp('^fizzbuzz ([\\d]+)$', 'i'));
+    const input = Aquarius.Triggers.messageTriggered(msg, /^fizzbuzz ([\\d]+)$/i);
```
Comment (IanMitchell) [inline src/commands/fizzbuzz.js]: \\d or \d?

File: src/commands/fizzbuzz.js (hunk 1 above)
Comment (kunaldes) [inline src/commands/fizzbuzz.js]: if I do that I get `falseBuzz` and `Fizzfalse`

But I put a ternary operator instead.

File: src/commands/fizzbuzz.js (hunk 4 above)
Comment (kunaldes) [inline src/commands/fizzbuzz.js]: Good catch.

I should write a test

but I'm not going to.

Comment (IanMitchell) [inline src/commands/fizzbuzz.js]: I haven't figured out a good way to setup a test framework for this yet haha. Most of the bugs come from discord.js API changes, and I'm not quite sure how to catch those

File: src/commands/fizzbuzz.js (hunk 3 above)
Comment (Mechdriver) [inline src/commands/fizzbuzz.js]: lol merge it

Comment (IanMitchell) [inline src/commands/fizzbuzz.js]: already in LOL

**Pull request merged**